lib/transmuxer/ts_transmuxer.js

```diff
@@ -881,6 +912,117 @@ shaka.transmuxer.TsTransmuxer = class {
       stream: stream,
     };
   }
+
+  /** @private */
+  aacTransmuxing_(uint8ArrayData, stream, reference, duration) {
```

Comment:
Could this delegate to `shaka.transmuxer.AacTransmuxer` instead to reduce duplication?

Reply:
I changed the entire implementation to reduce duplicate code.